frontend/actions/release.ts

```diff
@@ -90,3 +91,35 @@ export function deleteRelease(modelId: string, semver: string) {
     headers: { 'Content-Type': 'application/json' },
   })
 }
+
+export async function postSimpleFileForRelease(
+  modelId: string,
+  file: File,
+  onUploadProgress: (progress: AxiosProgressEvent) => void,
+  metadata?: string,
+) {
+  const fileResponse = await axios
+    .post(
+      metadata
+        ? `/api/v2/model/${modelId}/files/upload/simple?name=${file.name}&mime=${file.type}?${qs.stringify({
```

Comment:
I remember having issues with not all files having a `type`, so you might want to add back in the following logic from the old `postFile` function:
``` 
const mime = file.type || 'application/octet-stream'
```

You can then use `mime` in both of these urls.